dali/test/python/test_utils.py

```diff
@@ -76,41 +76,57 @@ def get_gpu_num():
     out_list = [elm for elm in out_list if len(elm) > 0]
     return len(out_list)
 
-def check_batch(batch1, batch2, batch_size, eps = 1e-07):
+
+def check_batch(batch1, batch2, batch_size, eps=1e-07, max_allowed_error=None):
+
+    def is_error(mean_err, max_err, eps, max_allowed_error):
```

Comment:
Shouldn't be:
```
        if max_allowed_error is not None and max_err > max_allowed_error:
            return True
        if mean_err > eps:
            return True
        return False
```

Reply:
Done

Reply:
Errm, or not done. I want to check the max_allowed_error if it's not None, otherwise use the previous behaviour - so the mean_err is checked only if the max_allowed_error in not None.

Reply:
Something like:
```
        if max_allowed_error is not None:
            if max_err > max_allowed_error:
                return True
        elif mean_err > eps:
            return True
        return False
```
may also work.

Reply:
So it is either max_allowed_error  or mean_err? Not both?

Reply:
Yes, I added a comment, as there is a lot of code that uses just the mean_error, but I want to use for max_error as well without rewriting this from scratch.